<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioTailTrimmer</class>
 <widget class="QMainWindow" name="AudioTailTrimmer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Tail Trimmer - made by feizhuliu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnImportFiles">
        <property name="text">
         <string>Import Audio Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAnalyzeTails">
        <property name="text">
         <string>Analyze Tails</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartTrim">
        <property name="text">
         <string>Start Trimming</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCutThreshold">
        <property name="text">
         <string>Trim Volume Threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbCutThreshold">
        <property name="suffix">
         <string>dB</string>
        </property>
        <property name="minimum">
         <number>-96</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>-48</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFadeDuration">
        <property name="text">
         <string>Fade out Duration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spbFadeDuration">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tblFileList">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <column>
       <property name="text">
        <string>Audio File</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tail Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
